shristy-kumari/html-css-task/style.css

```diff
@@ -0,0 +1,490 @@
+    display: flex;
+    flex-wrap: wrap;
+    gap: 20px;
+    margin: 0px auto;
+    padding-top: 10px;
+}
+
+.product-card{
+    width: 260px;
+    height: 200px;
+    background-color: #fff;
+    border-radius: 10px;
+    display: flex;
+    padding: 0px  10px;
+    color: #4b4a4a;
+    font-weight: 400;
+    margin: 10px 0px;
+}
+.product-text{
+    width: 100%;
+    padding: 10px 8px;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+}
+.product-name{
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 8px;
+}
+.product-info{
+    display: flex;
+    justify-content: space-between;
+}
+.left div,.right div{
+    margin-bottom: 8px;
+    font-size: 13px;
+}
+.head{
+    color: rgb(151, 150, 150);
+}
+
+/* sidebar styling here */
+
+#available,#notAvailable {
+    display: block;
+}
+
+.status-options{
+    display: flex;
+}
+
+.status-options,.color-options{
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    gap: 12px; 
+}
+
+.status-options input[type="radio"],.color-options input[type="checkbox"] {
+    margin: 0;
+    width: 16px;
+    height: 16px;
+}
+
+/* .status-options label {
+    font-size: 16px;
+    cursor: pointer;
+} */
+
+/* Sidebar Styling */
+
+#sidebar-overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: rgba(0, 0, 0, 0.5);
+    opacity: 0;
+    visibility: hidden; 
+    transition: opacity 0.4s ease-in-out;
+    z-index: 1000; 
+}
+
+.sidebar {
+    position: fixed;
+    top: 0;
+    right: -100%;
+    width: 350px;
+    height: 100%;
+    background: #edf3fc;
+    box-shadow: 0px 0 10px rgba(0, 0, 0, 0.2);
+    transition: right 0.4s ease-in-out;
+    padding: 20px;
+    overflow-y: auto;
+    z-index: 1001; 
+}
+
+#sidebar:target {
+    right: 0;
+}
+
+#sidebar:target ~ #sidebar-overlay {
+    opacity: 1;
+    visibility: visible;
+    transform: scale(1.2);
+    filter: (20%);
+}
+
+.sidebar h2 {
+    display: flex;
+    width: 100%;
+    font-size: 20px;
+    margin-bottom: 15px;
+    background-color: #fff;
+    padding: 10px 20px;
+}
+
+.form-group {
+    margin-bottom: 20px;
+    text-align: left;
+}
+
+.form-group label {
+    display: block;
+    margin-bottom: 5px;
+    font-size: 15px;
+}
+
+.form-group input, 
+.form-group textarea,
+.form-group select {
+    width: 100%;
+    padding: 8px;
+    border: 1px solid #fff;
+    border-radius: 5px;
+}
+
+.sidebar-buttons {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 20px;
+}
+
+.cancel-btn, .add-btn {
+    width: 48%;
+    padding: 10px;
+    border: none;
+    border-radius: 5px;
+    cursor: pointer;
+    font-size: 16px;
+    text-align: center;
+    display: inline-block;
+}
+
+.cancel-btn {
+    background: #fff;
+    text-decoration: none;
+    color: black;
+}
+
+.add-btn {
+    background: #182059;
+    color: white;
+}
+
+.clr-box{
+    width: 10px;
+    height: 10px;
+}
+.red{
```

Comment:
Give meaningful class names, looking at the class name, I can't guess its purpose.